### ISSUE TITLE
Dont know if issue or not supported

### ISSUE THREAD

**Issue by sucidaldeathtar** [EasyFarm/EasyFarm#219]
I use windower checked and I have all the files to run easyfarm but when casting say silence it will just keep casting it after the recast countdown.  Is this unsupported or is it pose to stop casting after the mob has been silenced?  if later  do I need to update a file maybe?

**Comment by Mykezero** [comment]
Yeah, there's no way for EF to tell if the spell has landed on an enemy. Eventually, this is something I would like to add to the program, but that would require parsing the chatlogs to find the player's spell has landed (Like how KParser use to determine that kind of stuff).
